invenio_app_ils/records_relations/api.py

```diff
@@ -192,6 +193,26 @@ def _build_sibling(self, related_pid, relation_type):
 
         return r
 
+    def _build_sequence(self, rec_pid, relation_type):
+        """Return the relation object with metadata."""
+        pid = rec_pid.pid_value
+        pid_type = rec_pid.pid_type
+
+        r = self._build_relations_object(pid, pid_type)
+
+        # retrieve any extra `relations_metadata`
+        from invenio_app_ils.records.api import IlsRecord
+        seq_record = IlsRecord.get_record_by_pid(pid, pid_type=pid_type)
+
+        metadata = RecordRelationsMetadata.get_metadata_for(
+            self.record, relation_type, pid, pid_type
+        )
+        r.update(metadata or {})
+        r["title"] = seq_record.get("title", "")
+        r["relation_type"] = relation_type
+
```

Comment:
@kprzerwa might appreciate extra metadata here. Language or edition? Actually, you should fix all of them and add the edition and the date. Library wants to see always the title, edition and date when showing a document, if set. This is because 2 docs could have the same title...

Reply:
I have added `title` and `edition`, publication year is not part of the series schema but I created a good first issue #736

Reply:
thanks